unit-tests-jest/numbers.test.js

```diff
@@ -0,0 +1,106 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers';
+
+describe('evenNumbers', () => {
+  it('returns the even numbers', () => {
+    const numbers = [1, 4, 5, 10, 0];
+    const result = evenNumbers(numbers);
+    expect(result).toStrictEqual([4, 10, 0]);
+  });
+  it('returns the even numbers', () => {
+    const numbers = [1, 4, 5, 10.2];
+    const result = evenNumbers(numbers);
+    expect(result).toStrictEqual([4]);
+  });
+  it('returns the even numbers and filter booleans', () => {
+    const numbers = [true, false, 4, 5, 10];
+    const result = evenNumbers(numbers);
+    expect(result).toStrictEqual([4, 10]);
+  });
+  it('returns the even numbers and filters falsey values', () => {
+    const numbers = [1, 4, 5, NaN, undefined, NaN];
+    const result = evenNumbers(numbers);
+    expect(result).toStrictEqual([4]);
+  });
+  it('returns the even numbers and filters strings', () => {
+    const numbers = [1, 4, 5, '10', 'hello'];
+    const result = evenNumbers(numbers);
+    expect(result).toStrictEqual([4]);
+  });
+});
+
+describe('toDollars', () => {
+  it('returns a number formatted to currency', () => {
+    const amount = 1.2;
+    const result = toDollars(amount);
+    expect(result).toMatch('$1.20');
```

Comment:
`toMatch` is used to test against a regular expression, but you are just testing against a string, so you could use `toEqual`.